src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: Efficiently load third-party JavaScript
+subhead: Avoid common pitfalls of using third-party scripts and improve load times and user experience.
+authors:
+  - mihajlija
+date: 2019-07-29
+description: |
+  Learn how to avoid common pitfalls of using third-party scripts and improve load times and user experience.
+hero: hero.jpg
+alt: Aerial view of shipping containers.  
+tags:
+  - post # post is a required tag for the article to show up in the blog.
```

Comment:
```suggestion
```
Given the third-party posts are kind of interdependent, I'm thinking maybe these shouldn't go on the blog. What do you think @robdodson?